### DISCUSSION TITLE
Cannot find shrinkwrap-resolver-depchain:jar

### DISCUSSION THREAD

**Discussion by ghost** [Minestom/Minestom#1358] [Help] [answered]
`Could not find artifact org.jboss.shrinkwrap.resolver:shrinkwrap-resolver-depchain:jar:3.1.4 in jitpack.io (https://jitpack.io)` getting this error on maven build tried to invalidate chaces changed maven version to 3.6.3 Iam clueless

**Comment by MrGazdag** [comment, marked as answer]
As noted [on the wiki](https://wiki.minestom.net/setup/dependencies):
> When using Maven it is recommended to exclude the artifact `shrinkwrap-resolver-depchain` from the group `org.jboss.shrinkwrap.resolver` as otherwise resolving the dependencies will fail. Shrinkwrap can be added as a separate dependency if needed without issues to restore its functionality.

This can be achieved by using the dependency declaration on the same wiki page instead: 
```xml
<dependencies>
    <!-- ... -->
    <dependency>
        <groupId>com.github.Minestom</groupId>
        <artifactId>Minestom</artifactId>
        <version>VERSION</version>
        <exclusions>
            <exclusion>
                <groupId>org.jboss.shrinkwrap.resolver</groupId>
                <artifactId>shrinkwrap-resolver-depchain</artifactId>
            </exclusion>
        </exclusions>
    </dependency>
</dependencies>
```
